### ISSUE TITLE
Calling path.bounds breaks geoAlbersUsa.fitSize.

### ISSUE THREAD

**Issue by gabrielflorit** [d3/d3-geo#59]
I think I've found an error dealing with `projection.fitSize` and `path.bounds`. Here's a block that reproduces it: https://bl.ocks.org/gabrielflorit/bb7ed6a56dbd5f66ee8c06b88a2b4f09

Specifically, accessing the `bounds` before calling `fitSize` makes the projection fit incorrectly:

```js
// Filter US to lower 48 + AK + HI
var conus = topojson.feature(us, {
	type: 'GeometryCollection',
	geometries: us.objects.states.geometries.filter(function (d) { return d.id < 60; }),
})

// Create projection
var projection = d3.geoAlbersUsa()

// Create path
var path = d3.geoPath().projection(projection)

// Comment-in the following line and the projection gets fitted
// incorrectly.
const bounds = path.bounds(conus)

// Fit projection in half the container.
projection.fitSize([width/2, height/2], conus)

svg.append('path')
	.datum(conus)
	.attr('d', path)
```

![screen shot 2016-09-20 at 3 31 57 pm](https://cloud.githubusercontent.com/assets/370976/18685780/6b436e34-7f47-11e6-946c-46396d2920c5.png)

**Comment by mbostock** [comment]
This problem appears to be specific to d3.geoAlbersUsa, because of the way it creates a “multiplexing” stream. It assumes the streams are independent, but in the case of the pathBounds stream, the state is global and so they clobber each other. I’ll need to rewrite pathBounds so that there can be multiple bounds streams.

**Comment by gabrielflorit** [comment]
Ah, interesting. Explains why reading pathBounds in a different scope didn't cause the error.

**Comment by mbostock** [comment]
Well it turns out the bug was slightly different, but a fix is incoming.

**Comment by mbostock** [comment]
Fixed in 1.2.5.

**Comment by gabrielflorit** [comment]
Wow, that was quick. Thank you so much!!!
